### PR TITLE
SASS-446 - Changed application directory structure to align with othe…

### PR DESCRIPTION
**Pull request by Dula-J**
…r itsass backends

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -3,13 +3,39 @@ import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
 
 val appName = "income-tax-employment"
 
-val silencerVersion = "1.7.3"
+val silencerVersion = "1.7.0"
```
Comment (BenCosfordHMRC) [inline build.sbt:6]: could this be 1.7.3?

File: build.sbt [hunk 2]
```diff
@@ -3,13 +3,39 @@ import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
...
+    ScoverageKeys.coverageHighlighting := true
+  )
+}
 
 lazy val microservice = Project(appName, file("."))
   .enablePlugins(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin)
   .settings(
     majorVersion                     := 0,
-    scalaVersion                     := "2.12.13",
+    scalaVersion                     := "2.12.12",
```
Comment (BenCosfordHMRC) [inline build.sbt:38]: could this be 2.12.13?

File: project/AppDependencies.scala [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
-    "com.vladsch.flexmark"    %  "flexmark-all"             % "0.36.8" % "test, it",
-    "org.scalatestplus.play"  %% "scalatestplus-play"       % "4.0.3"  % "test, it"
+    "uk.gov.hmrc"             %% "bootstrap-test-play-27"   % "3.4.0"                % Test,
+    "org.scalatest"           %% "scalatest"                % "3.0.9"                 % Test,
+    "com.typesafe.play"       %% "play-test"                % current                 % Test,
+    "org.pegdown"             %  "pegdown"                  % "1.6.0"                 % "test, it",
+    "com.vladsch.flexmark"    %  "flexmark-all"             % "0.35.10"               % "test, it",
+    "org.scalatestplus.play"  %% "scalatestplus-play"       % "4.0.3"                 % "test, it",
+    "com.github.tomakehurst"  %  "wiremock-jre8"            % "2.27.2"                % "test, it",
+    "org.scalamock"           %% "scalamock"                % "4.4.0"                 % Test
```
Comment (BenCosfordHMRC) [inline project/AppDependencies.scala:18]: could we see if there's any effort outside of simply bumping these values to what they were when we made the repo?

File: project/build.properties [hunk 4]
```diff
@@ -1 +1 @@
-sbt.version=1.4.8
+sbt.version=1.3.13
```
Comment (BenCosfordHMRC) [inline project/build.properties:1]: could this be 1.4.8?

File: project/AppDependencies.scala (hunk 3 above)
Comment (Dula-J) [reply project/AppDependencies.scala:18]: I've update everything except scalatest as it doesn't play nice with the current wiremockspec helper

File: build.sbt (hunk 2 above)
Comment (Dula-J) [reply build.sbt:38]: scoverage doesnt seem to work peroperly with 2.12.13

File: build.sbt (hunk 1 above)
Comment (Dula-J) [reply build.sbt:6]: 1.7.3 requires scala 2.12.13 which I'm not able to update as it does not work with scoverage

**Review by BenCosfordHMRC — APPROVED** [review]

**Pull request merged**